abb_irb4600_support/package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" ?>
-<package>
+<package="2">
```

Comment:
I believe the syntax is `<package format="2">`

http://docs.ros.org/jade/api/catkin/html/howto/format2/migrating_from_format_1.html#migrating-from-format1-to-format2

Reply:
agreed.  fixed in commit below.